core/src/main/java/io/undertow/UndertowMessages.java

```diff
@@ -623,4 +623,7 @@
 
     @Message(id = 199, value = "Read timed out after %s milliseconds.")
     ReadTimeoutException readTimedOut(long timeoutMilliseconds);
+
+    @Message(id = 200, value = "Out of flow control window: no WINDOW_UPDATE received from peer within %s miliseconds")
+    IOException noWindowUpdate(long timeoutMiliseconds);
 }
```

Comment:
It appears this message-id is not the same as [the PR for the master branch](https://github.com/undertow-io/undertow/pull/1192/files#diff-a503bdb32421a0a08548181d1aad3671f05712fadac9ddb0f732013e4163a0ef). 
I think the message-id should be consistent in every release.

Reply:
@msfm  thanks for pointing that out! I'll cherry-pick the fix from master to make sure they both contain the same ids.

Reply:
Okay, done, I cherry-picked the fix from master. :-)